app/scripts/modules/netflix/ci/ci.controller.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import _ from 'lodash';
+
+let angular = require('angular');
+
+require('./ci.less');
+
+module.exports = angular.module('spinnaker.netflix.ci.controller', [
+  require('../../core/authentication/authentication.service'),
+  require('../../core/application/service/applications.read.service'),
+  require('./build.read.service.js'),
+  require('angular-ui-router'),
+])
+  .controller('NetflixCiCtrl', function ($scope, authenticationService, app, buildService, $stateParams, $state) {
+    let attr = app.attributes;
+
+    this.viewState = { searchFilter: '', hasAllConfig: [attr.repoType, attr.repoProjectKey, attr.repoSlug].every((attr) => _.trim(attr)) };
+
+    this.getBuilds = () => {
+      if(!this.viewState.hasAllConfig) { return; }
```

Comment:
formatting is a little off here - add a space after `if` (same with four other spots in this PR)